.github/workflows/tests.yml

```diff
@@ -14,36 +14,49 @@ jobs:
     name: Integration tests
     runs-on: ubuntu-16.04
     steps:
-      - name: Checkout
-        uses: actions/checkout@v1
+      - name: Checkout code
+        uses: actions/checkout@v2
 
-      - uses: actions/setup-node@v1
+      - name: Set up node
+        uses: actions/setup-node@v2
         with:
-          node-version: '12'
-          always-auth: true
-          registry-url: https://registry.npmjs.org
+          node-version: 12
 
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
+      - name: Set output of cache
+        id: yarn-cache
```

Comment:
Minor:

```suggestion
        id: setup-yarn-cache
```

Reply:
I should have kept the CI out of this PR. I'm having issues with it, I'm in the process of simplifying and making it work. But maybe in hindsight, better 2 PRs

Reply:
Actually I'm struggling with Cypress running on Github (as u can tell for all my last commits)

I'll revert the original file and move this the CI part to another PR, wich by the way, I could use some help since it's too late for me and my head hurts. I'll explain what's the issue is...